src/Sarif/Autogenerated/SarifLogEqualityComparer.cs

```diff
@@ -10,8 +10,8 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// <summary>
     /// Defines methods to support the comparison of objects of type SarifLog for equality.
     /// </summary>
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.31.0.0")]
-    public sealed class SarifLogEqualityComparer : IEqualityComparer<SarifLog>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.32.0.0")]
+    internal sealed class SarifLogEqualityComparer : IEqualityComparer<SarifLog>
```

Comment:
Here's an example of how the files in the `NotYetGenerated` directory drifted out of sync from the generated files. When we made the equality comparers internal, we neglected to fix this one.